ui/app/AppLayouts/Onboarding/views/LoginView.qml

```diff
@@ -238,15 +242,18 @@ Item {
                     if (accountsPopup.opened) {
                         accountsPopup.close()
                     } else {
-                        accountsPopup.popup(userInfo, 0, userInfo.height+4)
+                        accountsPopup.popup(
+                                    userInfo,
+                                    (userInfo.width - accountsPopup.width) / 2,
+                                    userInfo.height + 8)
                     }
                 }
             }
 
             StatusPopupMenu {
                 id: accountsPopup
                 closePolicy: Popup.CloseOnEscape | Popup.CloseOnPressOutsideParent
-                width: parent.width
+                width: 346
```

Comment:
@faust4exe is it possible to avoid this fix weird number but rather user something like `width: parnet.width + 2 * Style.current.halfPadding`?